### PR TITLE
Change HTML format style

### PR DESCRIPTION
**Pull request by poznachowski**
Modified css style to made html format more concise. Tweak, when using full git commit message.

Overall, I find these small changes make the html looks a little better.

Sample output: http://s000.tinyupload.com/index.php?file_id=62694510974038154928

### CODE REVIEW

File: src/main/java/com/github/danielflower/mavenplugins/gitlog/renderers/SimpleHtmlRenderer.java [hunk 1]
```diff
@@ -73,11 +73,27 @@ public void renderCommit(RevCommit commit) throws IOException {
...
 				.append("</tr>").append(NEW_LINE);
 	}
 
+	private String formatLongMessage(String gitMessage) {
+		String lines[] = StringEscapeUtils.escapeHtml4(gitMessage).split(LINE_SEPARATOR);
+		if (lines.length <=1) {
+			return gitMessage;
+		}
+		StringBuilder builder = new StringBuilder();
+		for (int i = 0; i < lines.length; i++) {
```
Comment (danielflower) [inline src/main/java/com/github/danielflower/mavenplugins/gitlog/renderers/SimpleHtmlRenderer.java]: Would it be more straightforward to just wrap every line in a div: 

    "<div class=\"commitDetails\">" + lines[i] + "</div>"

Comment (poznachowski) [inline src/main/java/com/github/danielflower/mavenplugins/gitlog/renderers/SimpleHtmlRenderer.java]: I'm not sure if I get you. I want to emphasize first line of commit message (title), hence the if else part. First line does not have the 'commitDetails' class. That's how it looks like in the source:
``` html
<td>Enable to filter commits by a commiter name<br><span class="commitDetails"><br>Rationale: maven-jgitflow-plugin in contrary to maven-release-plugin does not provide any marker in the commit message.<br>It's not uncommon to have an artificial user handle project building and releasing (i.e. 'jenkins'). With this change, I can easily<br>filter commit messages by user name.<br></span></td>
```

**Comment by danielflower** [comment]
Thanks for the update. Apologies for my delay in dealing with this but I'm currently overseas. I'll let you know when it's released.

**Comment by danielflower** [comment]
Released in 1.11.0

**Pull request merged**